src/pages/Account/AccountContainer.jsx

```diff
@@ -13,53 +13,61 @@ class AccountContainer extends React.Component {
         'Organized all your contacts',
         'Contact DMV about updating tags',
         'Sent a request to your building manager',
-      ]
-    }
+      ],
+    };
   }
 
   /* istanbul ignore next */
   openConfirmModal = async () => {
-    const { actor, permission, isPageHidden } = this.props;
-    try {
-      const actions = [{
+    const { actor, permission, isPageHidden, setErrorState } = this.props;
+    const actions = [
+      {
         account: 'xtokens',
         name: 'transfer',
-        authorization: [{
-          actor: actor,
-          permission: permission
-        }],
+        authorization: [
+          {
+            actor: actor,
+            permission: permission,
+          },
+        ],
         data: {
-            from: actor,
-            to: ProtonSDK.requestAccount,
-            quantity: '5.000000 FOOBAR',
-            memo: 'Taskly'
-        }
-      }];
-      const tx = await ProtonSDK.sendTransaction(actions);
-      if (tx.processed.id) {
-        if (isPageHidden()) {
-          window.onfocus = this.loadTasksPage;
-        } else {
-          this.loadTasksPage();
-        }
+          from: actor,
+          to: ProtonSDK.requestAccount,
+          quantity: '5.000000 FOOBAR',
+          memo: 'Taskly',
+        },
+      },
+    ];
+    const { processed, error } = await ProtonSDK.sendTransaction(actions);
+    if (error) {
+      setErrorState(error);
```

Comment:
Let's use the early return pattern here as well, returning out of the method when there is an `error` and `processed && processed.id` is falsey.